### PR TITLE
Multiple entries in the AxisODF table were not deserialized correctly.

### PR DESCRIPTION
**Pull request by imikejackson**
Having more than a single entry in the Axis ODF table for Euler angles, Weights
and Sigmas would not correctly convert the Euler angle from radians to degrees.

Also applied some formatting and removed some variables that are never used or
referenced.

Signed-off-by: Michael Jackson <mike.jackson@bluequartz.net>

### CODE REVIEW

**Comment by imikejackson** [comment]
Since there is no unit test for this kind of failure I did a round trip from the pipeline file to the UI back to a pipeline file and back to the UI without any loss of precision. The code change now is a copy from the code that appears in a similar class StatsGenODFWidget.cpp.

File: Source/Plugins/SyntheticBuilding/Gui/Widgets/StatsGenAxisODFWidget.cpp [hunk 1]
```diff
@@ -619,29 +613,28 @@ void StatsGenAxisODFWidget::on_loadODFTextureBtn_clicked()
     return;
   }
 
-    size_t numOrients = 0;
-    QString filename = file;
-    std::ifstream inFile;
-    inFile.open(filename.toLatin1().data());
+  size_t numOrients = 0;
+  std::ifstream inFile;
+  inFile.open(file.toLatin1().data());
```
Comment (mmarineBlueQuartz) [inline Source/Plugins/SyntheticBuilding/Gui/Widgets/StatsGenAxisODFWidget.cpp:618]: You might want to look into the qPrintable for const char* conversion.  It does operate slightly differently as it doesn't force Latin1 characters, but if that is acceptable, I find it a lot more readable and easier to remember than whether I'm using toLatin1().data() or toLocal8Bit().data().  I wouldn't consider it important for right now, though.

**Review by mmarineBlueQuartz — APPROVED** [review]
This looks good.

File: Source/Plugins/SyntheticBuilding/Gui/Widgets/StatsGenAxisODFWidget.cpp (hunk 1 above)
Comment (imikejackson) [reply Source/Plugins/SyntheticBuilding/Gui/Widgets/StatsGenAxisODFWidget.cpp:618]: Thanks. will keep that in mind for future use.

**Pull request merged**